src/frontend/kjørelister/KjørelisteInnhold.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { useQuery } from '@tanstack/react-query';
+import { Routes } from 'react-router';
+import { Route, useParams } from 'react-router-dom';
+
+import { Alert, Loader, VStack } from '@navikt/ds-react';
+
+import { hentRammevedtak } from '../api/api';
+import { KjørelisteSkjema } from './components/Skjemaside/KjørelisteSkjema';
+import { KjørelisteProvider } from './KjørelisteContext';
+import { Kvitteringsside } from './components/Kvitteringsside/Kvitteringsside';
+import { Vedleggside } from './components/Vedleggside/Vedleggside';
+import { Oppsummeringsside } from './components/Oppsummering/Oppsummeringsside';
+
+export const KjørelisteInnhold = () => {
+    const kjørelisteId = useParams<{ kjorelisteId: string }>().kjorelisteId as string;
+
+    const { isPending, error, data } = useQuery({
```

Comment:
Liten quality of life her er å rename `data`, så er det tydeligere hva data er der den brukes
```suggestion
    const { isPending, error, data: rammevedtak } = useQuery({
```